### DISCUSSION TITLE
How to add radios to select them from the station radio selector?

### DISCUSSION THREAD

**Discussion by WrathPak** [wavelog/wavelog#1355] [Q&A] [answered]
I see this when logging a QSO - 
![image](https://github.com/user-attachments/assets/ea6e513f-f812-4185-a073-38a88b003600)

How does one add radios? I cannot find it anywhere.

**Comment by phl0** [comment, marked as answer]
> How does one add radios? I cannot find it anywhere.

It is auto-created as soon as you connet a radio via CAT/API.

**Reply by WrathPak** [reply]
ah, ok. I was hoping I could manually add and select radios to keep track of what radio was used. I think for now I will spilt up my stations to designate both the location and radio used to make that easier for me.

thanks for the response!

**Comment by HB9HIL** [comment]
https://github.com/wavelog/WaveLogGate

**Reply by DK5EW** [reply]
Is this only working if you use FLRIG ?
I connect my RIG via a 3.parity software (Win4Icom) and Gate is reading freq. perfect and transfer  to WL but I don`t see any RIG for choice.

**Reply by HB9HIL** [reply]
If Wavelog displays the current QRG in the QSO logging, you had to choose a radio beforehand in the second tab..
